### PR TITLE
Packed Bubble in a circle

### PR DESCRIPTION
**Pull request by shreshthmohan**
- [x] Search
- [x] Tooltip
- [x] y Value format and affixes
- [x] Validations
- [x] Options to expose for fine-tuning the shape

### CODE REVIEW

**Comment by shreshthmohan** [comment]
image

File: src/charts/packed_bubble/validateAndRender.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  // sizeValuePostfix = '',
+  sizeLegendGapInCircles: checkNumber,
+  // sizeLegendTitle = sizeField,
+  sizeLegendValues: checkNumericArray(),
+
+  // yValueFormat = '',
+  // yValuePrefix = '',
+  // yValuePostfix = '',
+
+  // searchInputClassNames = '',
```
Comment (plotthatstory) [inline src/charts/packed_bubble/validateAndRender.js:64]: Missing validations for force variable abstractions.

Rest all is fine. You can merge post validations.

**Review by plotthatstory — COMMENTED** [review]
Validations are missing for some options (force ones).

Rest all is good. Chart looks very good. 

Please merge post validations are done.

File: src/charts/packed_bubble/validateAndRender.js (hunk 1 above)
Comment (shreshthmohan) [reply src/charts/packed_bubble/validateAndRender.js:69]: Done merging.

**Pull request merged**